pkg/apiserver/event/event.go

```diff
@@ -57,6 +59,7 @@ func Register(r *gin.RouterGroup, s *Service) {
 	// TODO: add more api handlers
 	endpoint.GET("", s.listEvents)
 	endpoint.GET("/dry", s.listDryEvents)
+	endpoint.GET("/get", s.getEvent)
```

Comment:
I recommend following the restful API, use the format of `/events/id` to get the single event.